src/firefox/index.coffee

```diff
@@ -56,7 +56,6 @@ pageMod.PageMod {
 # codesy home page
 pageMod.PageMod {
   include : [
-    /.*localhost.*/
```

Comment:
Oops.  This one has to stay.  I haven't created a gulp task to morph javascript (yet).